packages/client/src/app/Main.jsx

```diff
@@ -57,6 +58,14 @@ for (const middleware of modules.middlewares) {
   });
 }
 
+for (const localization of modules.localizations) {
+  // eslint-disable-next-line
+  const resource = require(`@alienfast/i18next-loader!../modules/${localization.ns}/${localization.path}`);
```

Comment:
We do we do requires here, instead of doing them in their own modules?

Reply:
The idea was to rid modules of redundant logic in favor of merging all the files in one place. Probably this is not the best place for doing this, but language module is.

Reply:
@evgenijolshanskij Requires - are not a redundant logic. The require specifies what are the dependencies of the module

Reply:
@evgenijolshanskij We should avoid dynamic requires at all costs, they are poorly supported by various tools